fedify/vocab/lookup.ts

```diff
@@ -47,7 +47,12 @@ export interface LookupObjectOptions {
   tracerProvider?: TracerProvider;
 }
 
-const handleRegexp =
+/**
+ * Regular expression to match a fediverse handle in the format `@user@server` or `user@server`.
+ * The `user` part can contain alphanumeric characters and some special characters except `@`.
+ * The `server` part is all characters after the `@` symbol in the middle.
+ */
+export const handleRegexp =
   /^@?((?:[-A-Za-z0-9._~!$&'()*+,;=]|%[A-Fa-f0-9]{2})+)@([^@]+)$/;
```

Comment:
Hmm, I have doubts about whether exporting this is the right approach. Doing this would create another public API, but I'm not sure if that regex is really the API users want.

There seem to be two possible approaches:

1. Just accept the code duplication.

2. Add a proper new API like `parseFediverseHandle()`. The signature of this API would look something like:

    ```ts
    export interface FediverseHandle {
      readonly username: string;
      readonly host: string;
    }

    export function parseFediverseHandle(handle: string): FediverseHandle | undefined;
    ```

What do you think?

Reply:
If I accept duplications, I don't deserve to be called a programmer. I'll go with number 2.

Reply:
Changed at cd3035ec4623bd1b9e9b7040f3a8e6129669e6d7!
I additionally made useful functions to handle fediverse handles. For example, `convertFediverseHandle` can be applied in [line 133~135](https://github.com/fedify-dev/fedify/pull/278/files/c548281bb18aded21382e18fd17043e0786c85fa#diff-9167b739030a792c19d0315a534a2190801b87e479dc2b6bf8729250438bb2f3L133-L135).
However, the code file seems to be getting too long, so I think it would be better to split the file. Which path would be the best? Or just leave it as it is?